typhon/display.py

```diff
@@ -42,100 +58,141 @@ class TyphonDisplay(TyphonBase):
 
     parent: QWidget, optional
     """
-    def __init__(self, name=None, image=None, parent=None):
+    # Template types and defaults
+    Q_ENUMS(TemplateTypes)
```

Comment:
Is this why you need that confusing `TemplateTypes` class? Qt being weird

Reply:
Yes ... this is terrible, but neccesary

Reply:
That's awful